<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindowNew</class>
 <widget class="QMainWindow" name="LoginWindowNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelID">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="idEdit">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">qApp-&gt;setStyleSheet(&quot;QLineEdit { background-color: yellow }&quot;);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPIN">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>PIN</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pinEdit">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>Kirjaudu sisään</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelWelcome">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tervetuloa!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kirjaudu sisään korttisi ID-numerolla ja PIN-koodilla</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelinfo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>361</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelpic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Untitled.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>labelpic</zorder>
     <zorder>formLayoutWidget</zorder>
     <zorder>labelWelcome</zorder>
     <zorder>label</zorder>
     <zorder>labelinfo</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="labelErrorMsg">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>561</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>Tähän tulee virheilmoitus</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
